client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -292,11 +292,16 @@ def test_create_update_body(self):
 
         env_vars = {'CUDA_VISIBLE_DEVICES': "1,2", "VERTA_HOST": "app.verta.ai", "GIT_TERMINAL_PROMPT" : "1"}
 
-        parameter_json = endpoint._create_update_body(DirectUpdateStrategy(), resources, None, env_vars, 0)
-        assert parameter_json == {'build_id': 0, 'env': [{"name": 'CUDA_VISIBLE_DEVICES', 'value': '1,2'},
-                                                         {'name': 'GIT_TERMINAL_PROMPT', 'value': '1'},
-                                                         {"name": 'VERTA_HOST', 'value': 'app.verta.ai'}],
-                                  'resources': {'cpu_millis': 500, 'memory': '500Mi'}, 'strategy': 'rollout'}
+        parameter_json = endpoint._create_update_body(DirectUpdateStrategy(), resources, None, env_vars)
+        assert parameter_json == {
+            'env': [
+                {"name": 'CUDA_VISIBLE_DEVICES', 'value': '1,2'},
+                {'name': 'GIT_TERMINAL_PROMPT', 'value': '1'},
+                {"name": 'VERTA_HOST', 'value': 'app.verta.ai'}
+            ],
+            'resources': {'cpu_millis': 500, 'memory': '500Mi'},
+            'strategy': 'rollout',
+        }
```

Comment:
Removes `'build_id'`, and also reformats the dict in assert to be more standard.